pgcontents/checkpoints.py

```diff
@@ -65,7 +65,7 @@ def rename_checkpoint(self, checkpoint_id, old_path, new_path):
         """Rename a checkpoint from old_path to new_path."""
         with self.engine.begin() as db:
             return move_single_remote_checkpoint(
-                db, old_path, new_path, checkpoint_id,
+                db, self.user_id, old_path, new_path, checkpoint_id,
```

Comment:
@adzuci it gets called here: https://github.com/jupyter/notebook/blob/a6c5be5d60f122400b65d9c8d6488e78f25b9c2f/notebook/services/contents/manager.py#L234.  That should be exercised by https://github.com/ipython/ipython/blob/3.x/IPython/html/services/contents/tests/test_contents_api.py#L511, which is executed in our test suite for sure.